_posts/2018-02-19-propel2-wont-became-final-development-is-focused-on-propel3.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Propel2 won't became final, the development is focused on Propel3
```

Comment:
became -> become

Reply:
Thanks for finding this issue :-).
Sorry for the long replytime ... illness all over the world :(.